### ISSUE TITLE
Screen goes blank for 1 second

### ISSUE THREAD

**Issue by nbolton** [symless/synergy-core#2685]
**Imported issue:**
- Author: Rob Smallwood
- Date: 2010-10-26 09:39:13
- Legacy ID: 2741

**Steps to reproduce:**  
1. Move mouse to client screen, nothing unexpected, everything ok  
2. Move mouse from client to server screen. Server screen goes blank for 1 second then returns.  
3. <n/a>  

Expected: no flashing  
Actual: Server screen flashes off for 1 second, then returns.  

**Versions (Synergy, OS):**  

Server: Fedora core 13 64bit, synergy 1.3.4 on the server  
Client: Windows XP, synergy+ 1.3.4 on the client  

**Workaround:**  

No  

**Comments:**  

This prevents me from using synergys at all - too annoying

**Comment by the-wes** [comment]
Looks like this is still happening.

**Comment by nyetwurk** [comment]
In any case, disabling dpms doesn't seem like a good solution.

**Comment by ChristopherSchubert** [comment]
Confirming this is still occurring.  In this case `xset -dpms` does not resolve the issue.

Server OS:  Windows Server 2012 R2 Datacenter
Server Synergy: 1.8.2-stable-36cd521

Client OS:  Debian GNU/Linux 8
Client Synergy: 1.8.2-stable-36cd521

**Comment by nlyan** [comment]
@ChristopherSchubert what desktop environment are you using on Debian?

**Comment by ChristopherSchubert** [comment]
@nlyan Gnome 3.14.1

**Comment by qizongjun** [comment]
I am seeing this issue too with Synergy server running on Ubuntu 14.04 LTS and a Windows laptop as client.

The moment I try to move back mouse from Windows to Linux, the Linux screen turn dark, and each time, I need to pull mouse back to windows and **hit the Linux screen exactly twice** for a successful landing to Linux screen.

My synergy version is 
➜  ~ synergys --version
synergys 1.8.6, protocol version 1.6

**Comment by DerekVeit** [comment]
I solved this by just commenting out `screensaver(false);` at the top of `XWindowsScreen::enter()` in src/lib/platform/XWindowsScreen.cpp and recompiling.

It looks like something in the deactivation of the screen saver causes it to be momentarily activated.  I don't use a screen saver, so I never need it to be deactivated.  The really old version of Synergy that I used before, 1.3.1, did not have that line or this bug, and this bug has been reported for versions as far back as 1.3.4, which did already have that line.

The `xset -dpms` workaround did not work for me.
